docs/install/install.md

```diff
@@ -21,21 +21,23 @@ limitations under the License.
 
 
 
-## From binary package
+## Zeppelin Installation
+Welcome to your first trial to explore Zeppelin ! 
 
-   Download latest binary package from [Download](../download.html).
+In this documentation, we will guide you how you can install Zeppelin from **Binary Package** or build from **Source** by yourself. Plus, you can get a specific infomation about Zeppelin configurations at the below **Zeppelin Configuration** section.
 
+### Install with Binary Package
 
-## Build from source
+If you want to install Zeppelin with latest binary package, please visit [this page](http://zeppelin.incubator.apache.org/download.html).
 
-   Check instructions in [README](https://github.com/apache/incubator-zeppelin/blob/master/README.md) to build from source.
+### Build from Zeppelin Source
 
+You can also build Zeppelin from the source. Please check instructions in `README.md` in [Zeppelin github](https://github.com/apache/incubator-zeppelin/blob/master/README.md). 
 
 
-## Configure
-
-Configuration can be done by both environment variable(conf/zeppelin-env.sh) and java properties(conf/zeppelin-site.xml). If both defined, environment vaiable is used.
+## Zeppelin Configuration
 
+You can configure Zeppelin with both environment variables in `conf/zeppelin-env.sh` and java properties in `conf/zeppelin-site.xml`. If both are defined, then the environment variable will be used priorly.
```

Comment:
Better have **environment variables** and **java properties** bold

We could probably have a better wording here, currently it can mean multiple things.
@Leemoonsoo If we have both configuration files, would it use both and give `conf/zeppelin-env.sh`'s values priority in case of overlap? Or would it only use `conf/zeppelin-env.sh` file?